3rdparty/wxWidgets_setup_h/wx/setup.h

```diff
@@ -44,7 +44,7 @@
 // in the version after it completely.
 //
 // Recommended setting: 0 (please update your code)
-#define WXWIN_COMPATIBILITY_3_0 0
+#define WXWIN_COMPATIBILITY_3_0 1
```

Comment:
Need this one because these 3 functions are deprecated
https://github.com/vslavik/winsparkle/blob/7ca9b991b632fa191d07ed18cd61a2f55a4c1675/src/ui.cpp#L116-L118

Reply:
Turned out to be straightforward, fortunately:  [81e5fe3](https://github.com/vslavik/winsparkle/pull/249/commits/81e5fe3348be45f915098d252693f58ad44e9267)